x-pack/plugins/security_solution/public/resolver/view/index.tsx

```diff
@@ -28,6 +29,11 @@ export const Resolver = React.memo(function ({
    * Used as the origin of the Resolver graph.
    */
   databaseDocumentID?: string;
+  /**
+   * A string literal describing where in the app resolver is located,
+   * used to prevent collisions in things like query params
+   */
+  documentLocation: string;
```

Comment:
:grey_question:  defer for now, but something like  `Pick<TimelineModel,'id'>` might be a little more descriptive/helpful here.

Reply:
...but maybe also too narrow, carry on actually.